spec/models/task_spec.rb

```diff
@@ -4,30 +4,33 @@
   let(:title) { "Short text" }
   let(:description) { "Long text" }
   let(:due_date) { "11-11-1999" } 
-  let(:status) { true }
+  let(:done) { true }
 
   context "valid params" do
```

Comment:
`context` description should start with word `with` or `when`.

http://betterspecs.org/#contexts